articles/20241126_Using_Environmental_Variables_in_Daytona.md

```diff
@@ -0,0 +1,134 @@
+
+This article will explain using the `daytona env` command to set and manage environmental variables within Daytona workspaces. You will learn how to set variables, create a workspace, and access these variables within a workspace.
+
+### TL;DR
+
+- **What**: Learn to use `daytona env set` for managing workspace environmental variables
+
+- **Why**: Simplify configuration and securely manage development environment settings
+
+- **How**:
+  - Set variables using `daytona env set key=value`
+  - Create a workspace with `daytona create --code`
+  - Verify variables with `daytona env list`
+
+## Prerequisites
+
+- Daytona installed on your system
+- Docker
+- Visual Studio Code (VSCode)
+
+## Set Environmental Variables
+
+Before adding an environment variable to Daytona, ensure that Docker and your server are running. Start your server using:
+
+```bash
+daytona server
+```
+
+Then use `daytona env set` to define environmental variables:
+
+```bash
+daytona env set USERNAME=JohnDoe PASSWORD=123456
+```
+
+This command allows you to set multiple environmental variables in a single operation.
+
+### How `daytona env set` Works
+
+The command stores environment variables at the workspace level. The variables are persistent across multiple workspaces and are accessible to all containers and development tools within the workspace. This provides a secure way to manage configuration without modifying project files.
+
+You can verify that the environmental variables have been saved by running:
+
+```bash
+daytona env list
+```
+
+You should see an output similar to the image below:
+
+![Screenshot example of Daytona env variable list](assets/20241126_Using_Environmental_Variables_in_Daytona_1.png)
+
+## Create a New Workspace
+
+You can create and open a new workspace using Daytona's create command:
+
+```bash
+daytona create https://github.com/bellatrick/python_starter.git
+```
+
+This command will clone the repository, set up the development environment, and open Visual Studio Code. The `python_starter` repository is a simple Python project pre-configured with a Dev Container.
+
+### Using Environmental Variables in Your Project
+
+In your development environment, create a Python script to demonstrate accessing environment variables:
+
+`env_demo.py`:
+
+```python
+import os
+
+def main():
+    # Access environmental variables
+    username = os.environ.get('USERNAME')
+    password = os.environ.get('PASSWORD')
+
+    if username and password:
+        print(f"Logged in as: {username}")
+        # Note: In a real application, never print passwords!
+    else:
+        print("Environmental variables not found")
+
+if __name__ == "__main__":
+    main()
+```
+
+Run the script using:
+
+```bash
+python env-demo.py
+```
+
+Your output should look like this:
+
+![Result of running python script](assets/20241126_Using_Environmental_Variables_in_Daytona_2.png)
+
+### Best Practices
+
```

Comment:
add one small intro sentence to section

Reply:
Hey @nkkko
I've made all the suggested changes.